Styles.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+
+body {
+  margin-right: 486.5px;
+  margin-left: 486.5px;
+}
+
+.header_section {
+  background-color: #3c3a39;
+  height: 92px;
+  position: relative;
+  display: flex;
+}
+
+#unicon {
+  position: absolute;
+  inset: 63px 19px 16px 323px;
+}
+
+.headline_section {
+  background-color: #1c1a19;
+  background-image: url(Sources/Group.png);
+  background-repeat: no-repeat;
+  background-position: 244px 322px;
+  height: 590px;
+  position: relative;
+  display: flex;
+}
+
+#name {
+  color: white;
+  position: absolute;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 19px;
+  text-decoration: none;
+  text-align: center;
+  inset: 56px 240px 12px 24px;
+}
+
+#bkg_illustrations {
+  position: absolute;
+  inset: 316px 0 0 243px;
+}
+
+.social_media {
+  position: absolute;
+  inset: 430px 0 0 24px;
+  list-style: none;
+}
+
+.social_media li {
+  display: inline;
+}
+
+#primary_text {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  position: absolute;
+  inset: 120px 0 0 24px;
+}
+
+#supporting_text {
+  color: #fff5e1;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  position: absolute;
+  inset: 256px 0 0 24px;
+}
+
+#description {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] Kindly make sure you import the font from google Fonts in the right way as you can see here the error attached to the bad usage of `font` from google.
`hint` `After selecting the font you want to use on google make sure you copy the link provided and paste it into your HTML header or import it to the top of your CSS file. `
![Screenshot from 2022-09-23 09-12-38](https://user-images.githubusercontent.com/97100091/191910033-36580334-442c-4c4c-98a5-812bdff27174.png)